sktime/transformations/series/tests/test_window_summarizer.py

```diff
@@ -94,6 +102,13 @@ def count_gt100(x):
 @pytest.mark.parametrize(
     "kwargs, column_names, y, target_cols, truncate",
     [
+        (
```

Comment:
I would like to see that this test is also executed with a dataFrame, with multi index data and hierarchical data.